golem/managers/activity/pool.py

```diff
@@ -13,7 +12,22 @@
 logger = logging.getLogger(__name__)
 
 
-class ActivityPoolManager(BackgroundLoopMixin, ActivityPrepareReleaseMixin, ActivityManager):
+@Activity.register
+class PoolActivity(ActivityWrapperMixin):
+    def __init__(
+        self,
+        put_activity_in_pool_func,
```

Comment:
It's better to keep order of the base class, but with one additional keyword arg.